src/mame/drivers/missb2.cpp

```diff
@@ -16,15 +16,13 @@ written, so it may be normal behaviour.
 
 #include "emu.h"
 #include "includes/bublbobl.h"
-
 #include "cpu/z80/z80.h"
 #include "sound/3526intf.h"
 #include "sound/okim6295.h"
 #include "machine/watchdog.h"
 #include "screen.h"
 #include "speaker.h"
 
-
```

Comment:
The two blank lines between logical sections (in this case includes and type definitions) is a useful visual guide.  It's not like it's half a dozen blank lines in a row.